### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by Drauboss**
German Translation. I have tried testing it before by replacing the old localize file with the new (german translation added) file in the plex_assistant folder. But i got an error on Home Assistant that it couldnt load the integration.

### CODE REVIEW

**Comment by maykar** [comment]
Test with the changes I've made and let me know how it works for you.

File: custom_components/plex_assistant/localize.py [hunk 1]
```diff
@@ -955,57 +953,97 @@
...
+            ],
         },
         "latest": {
             "keywords": [
-                  "neuste", 
-                  "aktuellste", 
-                  "letzte", 
-                  "aktuelle"
-              ],
-            "pre": ["die"]
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py]: A comma was removed here after `["die"]`. This is enough to cause errors. Found a few things like this. They are much easier to see if you stick to the formatting of the template.

**Comment by Drauboss** [comment]
It works now. Thank you. But i got a problem with playing unwatched episodes of a tv show. For example: im currently on epsiode 9 of Dragonball but when i say "play the next episode of dragonball"(in german) it always plays the first episode. The same thing happens when using the english version of your integration.

**Comment by maykar** [comment]
That's a current issue to be fixed soon, technically "next" isn't really a command yet, but it's being worked on.

https://github.com/maykar/plex_assistant/issues/36#issuecomment-687663176

I'll try to get this in the next release

**Pull request merged**